OSDOCS-6661: Updates to MicroShift manifests (#63830)
openshift/openshift-docs: modules/microshift-manifests-override-paths.adoc

// Module included in the following assemblies:
//
// * microshift//running_applications/microshift-applications.adoc

:_content-type: PROCEDURE
[id="microshift-manifests-override-paths_{context}"]
= Override the list of manifest paths

> You can override the list of default paths with the following procedure.

Reviewer: ```suggestion
You can override the list of default manifest paths by using a new single path, or by using a new glob pattern for multiple files. Use the following procedure to customize your manifest paths.
```
Something like that?

Author: I like it. I had that as a placeholder, but I like yours better than what I had going on in my head.

.Procedure

> . You can set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++` in the configuration file for a single path. Alternativly, you can set `kustomizePaths` to `++"++/opt/alternative/path.d/++*"++` in the configuration file for a glob pattern.

Reviewer 2: ```suggestion
. You can set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++` in the configuration file for a single path. Alternatively, you can set `kustomizePaths` to `++"++/opt/alternative/path.d/++*"++` in the configuration file for a glob pattern.
```

> . You can set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++` in the configuration file for a single path. Alternatively, you can set `kustomizePaths` to `++"++/opt/alternative/path.d/++*"++` in the configuration file for a glob pattern.

Reviewer: ```suggestion
. You can set `manifests.kustomizePaths` to `++"++</opt/alternate/path>++"++` in the configuration file for a single path. Alternatively, you can set `kustomizePaths` to `++"++</opt/alternative/path.d/>++*"++` in the configuration file for a glob pattern.
```
I'm not sure which parts of that entry are required, and which are user-specific, but we usually <user-input> format that.

Author: Thanks @ShaunaDiaz i was forgot about that as an option.

> . You can set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++` in the configuration file for a single path. Alternatively, you can set `kustomizePaths` to `++"++/opt/alternative/path.d/++*"++` in the configuration file for a glob pattern.

Reviewer: ```suggestion
. You can set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++` in the configuration file for a single path. Alternatively, you can set `kustomizePaths` to `++"++/opt/alternative/path.d/++*"++` in the configuration file for a glob pattern.
```
I might break this into two bullets under the heading you have as the intro:

. Override the list of default paths by inserting your own values and running one of following commands:

. . Set `manifests.kustomizePaths` to `<++"++/opt/alternate/path++"++>` in the configuration file for a single path.

. .  Set `kustomizePaths` to `++"++</opt/alternative/path.d/>++*"++` in the configuration file for a glob pattern.

I assume you can only do one or the other? If both, we should create the third option.

+
[source,terminal]
----
manifests:
    kustomizePaths:
        - <location> <1>
----

> <1> Set the location to `++"++/opt/alternate/path++"++` for single path and `++"++/opt/alternative/path.d/++*"++` for a glob pattern.

Reviewer 2: ```suggestion
<1> Set each location entry to an exact path (`++"++/opt/alternate/path++"++`) or a glob pattern (`++"++/opt/alternative/path.d/++*"++`).
```

I'm sure that doesn't fit our style guide, but I hope you get the idea.

Author: Yeah, the style guide isn't a big fan of the use of parenthesis. I did update to "using".

> <1> Set each location entry to an exact path using `++"++/opt/alternate/path++"++` or a glob pattern using `++"++/opt/alternative/path.d/++*"++`.

Reviewer 2: This still reads as those these are actual values to use, but they're just examples of paths or patterns.

Reviewer: See above some ideas.

// Module included in the following assemblies:
//
// * microshift//running_applications/microshift-applications.adoc

:_content-type: PROCEDURE
[id="microshift-manifests-override-paths_{context}"]
= Override the list of manifest paths
You can override the list of default paths with the following procedure.

.Procedure

. In the configuration file, set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++`.
+
[source,terminal]
----
manifests:
    kustomizePaths:
        - "/opt/alternate/path"
----

> . The values of `kustomizePaths` may be glob patterns.

Author: I am thinking maybe this should be a note nested under the first step, but I am not sure.

Reviewer 2: Yeah, you could combine the 2 examples and point out that a value can be a single path or glob pattern.

+
[source,terminal]
----
manifests:
    kustomizePaths:
        - "/opt/alternative/path.d/*"
----

> . To disable loading manifests, set the configuration option to an empty list.

Reviewer 2: Somewhere near here we should have a note/warning that the configuration file overrides the defaults entirely, so if the kustomizePaths value is set to _anything_, only the values in the configuration file are used. That's why setting the value to an empty list disables manifest loading.

// Module included in the following assemblies:
//
// * microshift//running_applications/microshift-applications.adoc

:_content-type: PROCEDURE
[id="microshift-manifests-override-paths_{context}"]
= Override the list of manifest paths
You can override the list of default manifest paths by using a new single path, or by using a new glob pattern for multiple files. Use the following procedure to customize your manifest paths.

.Procedure

. Override the list of default paths by inserting your own values and running one of the following commands:

.. Set `manifests.kustomizePaths` to `<++"++/opt/alternate/path++"++>` in the configuration file for a single path.

.. Set `kustomizePaths` to `,++"++/opt/alternative/path.d/++*"++.` in the configuration file for a glob pattern.
+
[source,terminal]
----
manifests:
    kustomizePaths:
        - <location> <1>
----

> <1> Set each location entry to an exact path using `++"++/opt/alternate/path++"++` or a glob pattern using `++"++/opt/alternative/path.d/++*"++`.

Reviewer 3: ```suggestion
<1> Set each location entry to an exact path by using `++"++/opt/alternate/path++"++` or a glob pattern by using `++"++/opt/alternative/path.d/++*"++`.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u
